Command/PrintIndicesCommand.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Apisearch Bundle.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ */
+
+declare(strict_types=1);
+
+namespace Apisearch\Command;
+
+use Apisearch\Model\Index;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class PrintIndicesCommand.
+ */
+class PrintIndicesCommand extends WithRepositoryBucketCommand
+{
+    /**
+     * Configures the current command.
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('apisearch:print-indices')
+            ->setDescription('Print all indices')
+            ->addArgument(
+                'repository',
+                InputArgument::REQUIRED,
+                'Repository name'
+            )
+            ->addOption(
+                'app_id',
+                'a',
+                InputOption::VALUE_OPTIONAL,
+                'AppId to filter indices',
+                null
+            );
+    }
+
+    /**
+     * Dispatch domain event.
+     *
+     * @param InputInterface  $input
+     * @param OutputInterface $output
+     *
+     * @return mixed|null
+     */
+    protected function runCommand(InputInterface $input, OutputInterface $output)
+    {
+        $repository = $input->getArgument('repository');
+        $appId = $input->getOption('app_id');
+        $indices = $this
+            ->repositoryBucket->findRepository($repository)
+            ->getIndices($appId);
+
+        /**
+         * @var Index
```

Comment:
I think this should be `@var Index $index` !? Personally I would also move it down to line 73 or 74.

Reply:
`composer fix-code` removes `$index` from there, interestingly. And PrintTokensCommand has a comment on the same position. :) I compare from there.

Reply:
@hkulekci I think, You should use this command locally, and push fixed code to the repository. This command is used to standardize Your code to other parts of app and should be available in an improved form :)